### DISCUSSION TITLE
how to open neotree window to the right?

### DISCUSSION THREAD

**Discussion by navxio** [LazyVim/LazyVim#1005] [Q&A] [answered]
`<leader>e` default window

**Comment by dpetka2001** [comment, marked as answer]
```lua
{
   "nvim-neo-tree/neo-tree.nvim",
   opts = {
      window = {
         position = "right",
      },
   },
}
```
should do it.

**Reply by navxio** [reply]
@dpetka2001 so the custom ops are merged with the default ops, and can overwrite the default values, right?

**Reply by dpetka2001** [reply]
Yes, it is also mentioned in the [docs](https://www.lazyvim.org/plugins#%EF%B8%8F-customizing-plugin-specs). You should take some time to read through them.

**Reply by navxio** [reply]
@dpetka2001 gosh I missed that plugin page, didn't know that existed because I'd just use the arrow to expand the plugin list.. can we do something about it?

**Reply by Keshav-writes-code** [reply]
where to put this config snippet?

**Reply by dpetka2001** [reply]
It's mentioned in the [docs](https://www.lazyvim.org/configuration) where to put your personal configuration for plugins. 

For example, you create a file `~/.config/nvim/lua/plugins/neotree.lua` with contents
```lua
return {
{
   "nvim-neo-tree/neo-tree.nvim",
   opts = {
      window = {
         position = "right",
      },
   },
},
}
```

**Reply by sefatanam** [reply]
I tried but not works in v14.14

**Reply by jonnjonnjo** [reply]
I haven't found a permanent solution by changing the *.lua, but you could temporarily move or configure the Explorere Windows through <C-w> in normal mode and you could change the position of the explorer
<img width="1920" height="1200" alt="image" src="https://github.com/user-attachments/assets/9ecea0a4-def3-4802-a8b5-a653bcb9e588" />

From the [man page](https://www.lazyvim.org/plugins/treesitter#nvim-treesitter), the new version of LazyVim uses nvim-treesitter instead of neo-tree. 

I tried to ask it using Claude & Gemini, but both LLM are too fucking retarded enough that they first show the wrong solution (showing the neo-tree solution) and then hallucinates/made up their own answer.

**Reply by jonnjonnjo** [reply]
Sorry, you could modify the window by choosing <leader>w instead :D

**Reply by walleXD** [reply]
this worked for me:
```lua
return {
  "folke/snacks.nvim",
  opts = {
    explorer = {
      replace_netrw = true,
    },
    picker = {
      sources = {
        explorer = {
          layout = {
            layout = {
              position = "right",
            },
          },
        },
      },
    },
  },
}
```
